build/ansible/pmm-docker/post-build.yml

```diff
@@ -129,4 +129,11 @@
         group: root
         state: file
 
+    - name: Set ownership for pmm-agent config file
```

Comment:
I wonder whether this'd better be done by changing `PMM_GROUP` to `root` instead of `pmm`, which most likely leads to improper ownership of `pmm-agent.yaml`.

See [here](https://github.com/percona/pmm/blob/11355689ecfbbed8da247b5ecf1b7b89be9c1160/build/ansible/roles/pmm-client/tasks/main.yml#L25).

Reply:
Good point, fixed